packages/atomic/src/components/recommendations/atomic-recs-interface/atomic-recs-interface.tsx

```diff
@@ -55,10 +56,19 @@ export class AtomicRecsInterface
 
   @State() public error?: Error;
 
+  /**
+   * The recommendation interface [query pipeline](https://docs.coveo.com/en/180/).
+   *
+   * If the recommendation interface is initialized with `initializeWithSearchEngine`, the query pipeline should instead be configured on the engine.
```

Comment:
Maybe a link would help:
```suggestion
   * If the recommendation interface is initialized using [`initializeWithRecommendationEngine`](https://docs.coveo.com/en/atomic/latest/reference/recommendation-components/atomic-recs-interface/#initializewithrecommendationengine), the query pipeline should instead be configured in the target engine.
```